### PR TITLE
Customizing a router

### PR DESCRIPTION
**Pull request by pweil-**
Example of customizing a router to use stick-tables

@adellape @rajatchopra

### CODE REVIEW

File: admin_guide/router.adoc [hunk 1]
```diff
@@ -54,3 +54,91 @@ connecting to back-ends for re-encrypt terminated routes are stored in the
...
+HAProxy configuration file.  If you would like to customize a router to meet
+your needs you are able to change the template file, build a new docker image,
+and run a customized router.
+
+One common case for this may be implementing new features withing the
+application backends.  For instance, it might be desirable in a highly available
+setup to use stick-tables that synchronizes between peers.  The router
+plugin provides all the facilities necessary to make this customization.
+
+.Using Stick Tables
```
Comment (adellape) [inline admin_guide/router.adoc]: Is this meant to be a heading?

**Comment by adellape** [comment]
@pweil- Thanks, one comment inline. Will wait for @rajatchopra feedback.

BTW, after merging this, I'd like to move the CLI commands for actually creating the router from here...

http://docs.openshift.org/latest/architecture/core_objects/routing.html#routers

...into this Admin Guide topic (with a link from that Arch topic). Unless you feel otherwise.

**Comment by rajatchopra** [comment]
LGTM

File: admin_guide/router.adoc (hunk 1 above)
Comment (pweil-) [inline admin_guide/router.adoc]: It was meant to mark the start of the example customization of the router.  Didn't really think it should be a clickable part of the TOC but I have no preference either way.

**Comment by pweil-** [comment]
> BTW, after merging this, I'd like to move the CLI commands for actually creating the router from here...
> 
> http://docs.openshift.org/latest/architecture/core_objects/routing.html#routers
> 
> ...into this Admin Guide topic (with a link from that Arch topic). Unless you feel otherwise.

I feel that you know best!

**Comment by adellape** [comment]
Cool, thanks, all. Merging.

**Pull request merged**